src/main/java/com/checkmarx/sdk/config/ScaProperties.java

```diff
@@ -29,4 +29,5 @@
     private String username;
     private String password;
     private boolean enabledZipScan;
+    private String team;
```

Comment:
I would call this property something like `teamForNewProjects`, which is self-documenting.

Reply:
Actually, I picked the 'team' name to be consistented with our existing 'team' property on our 'checkmarx' configuration section

Reply:
These 2 properties behave differently (as seen in com.checkmarx.sdk.service.CxService#updateProjectDetails). Giving them the same name may cause the user to make wrong assumptions and cause confusion.

Reply:
Updated.